Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -241,7 +243,7 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
         }
       }
 
-    self.contentInset = self.viewWillAppearProperty.signal
+    self.contentInset = self.viewWillAppearIsEditorialProperty.signal.filter(isFalse)
```

Comment:
Does updating the content inset actually break anything?

Reply:
It is updated by the parent when contained in editorial VC.